api/ccapi/include/model.h

```diff
@@ -40,6 +41,30 @@ enum class ModelType {
   UNKNOWN     /** Unknown */
 };
 
+/**
+ * @brief Model saving options
+ *
+ */
+enum class ModelSaveLoadFlags {
+  MODEL_SAVE_LOAD_FLAGS_INFERENCE_PARAMS =
```

Comment:
I think we need check again how these options will work. Cause once we set these enum, It's going to be difficult to change.

Reply:
As per offline discussion, unused enum for now will be commented